.github/actions/generate-coverage/scripts/install_cargo_nextest.py

```
@@ -63,62 +104,70 @@ def _normalize_machine(machine: str) -> str:
 def _report_platform_key(key: str) -> None:
-    """Report the selected cargo-nextest platform key."""
-    if key == "linux-x86_64-musl":
-        typer.echo("Detected libc for cargo-nextest: musl")
-    elif key == "linux-x86_64-gnu":
-        typer.echo("Detected libc for cargo-nextest: glibc")
-    typer.echo(f"Selected cargo-nextest platform key: {key}")
+    """Log the normalized checksum lookup key."""
+    logger.info("Resolved cargo-nextest platform key: %s", key)
+
+
+class _ExpectedShaResult(tuple[str, str | None]):
```

Ruff (0.15.17)

[warning] Object does not implement `__hash__` method

(PLW1641)